core/src/main/resources/application.yaml

```diff
@@ -1,4 +1,68 @@
-#file: noinspection SpringBootApplicationYaml
+app:
+  advisor:
+    logger:
+      order: 10
+    chat-memory:
+      order: 20
+      memory-size: 50
+      default-system-advise-text: "
+                        Use the conversation memory from the MEMORY section, surrounded by --------------------- to provide accurate answers.
+                        ---------------------
+                        
+                        MEMORY:
+                         {memory}
+                         
+                        --------------------- "
+      default-system-advise-text-ru: "
+                              Используй память разговора из раздела MEMORY, окруженную ---------------------, чтобы давать точные ответы.
+                              ---------------------
+                              
+                              MEMORY:
+                               {memory}
+                               
+                              --------------------- "
+    rag:
+      default-user-text-advice: "
+      Context information is below, surrounded by ---------------------
```

Comment:
Use the following pieces of context to answer the question at the end.
{context}
mb this should work better